server/src/graphql/mutations/createAppGithub.ts

```diff
@@ -34,37 +40,38 @@ export const createAppGithub: MutationResolvers['createAppGithub'] = async (
 
   const octokit = new Octokit({});
 
-  let repo;
-
   const repoData = getRepoData(input.gitRepoUrl);
-  try {
-    repo = await octokit.repos.get({
-      owner: repoData.owner,
-      repo: repoData.repoName,
-    });
-  } catch (error) {
-    console.log(error);
+
+  const repo = await octokit.repos.get({
```

Comment:
if repo does not exist what will happen here?

Reply:
@pradel after latest changes it will throw error.